File: Ruby/7kyu/multiply_the_strings_in_the_array.rb
# You received an array with two strings. Create a function that will return their product as a string.
#
# E.g. arrMultiply(['9','6']) should return '54'

# def arrMultiply(arr)
#   # your solution here
# end

# My solution

def arrMultiply(arr)
  (arr[0].to_i * arr[1].to_i).to_s
end

# The most popular solution

# def arrMultiply(arr)
#   arr.map(&:to_i).reduce(:*).to_s
# end

File: JavaScript/7kyu/multiplyTheStringsInTheArray.js
// You received an array with two strings. Create a function that will return their product as a string. E.g.
//
// arrMultiply(['9','6']) should return '54'
//
// function arrMultiply(arr){
//    //your solution
// }

// My solution

function arrMultiply(arr){
   return (arr[0] * arr[1]).toString();
}

// The most popular solution
// 
// let arrMultiply = ([a, b]) => '' + (a * b);
